### PR TITLE
download cni plugin from gcs bucket

### PR DESCRIPTION
**Pull request by yliaog**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
switched to download cni plugins from github to gcs bucket

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 1]
```diff
@@ -851,18 +851,18 @@ Import-Module -Name $modulePath'.replace('K8S_DIR', ${env:K8S_DIR})
...
-  $CNI_RELEASE_VERSION = 'v0.8.0'
+  $CNI_RELEASE_VERSION = 'v0.8.2-gke.0'
   if ((ShouldWrite-File ${env:CNI_DIR}\win-bridge.exe) -or
       (ShouldWrite-File ${env:CNI_DIR}\host-local.exe)) {
     $tmp_dir = 'C:\cni_tmp'
     New-Item $tmp_dir -ItemType 'directory' -Force | Out-Null
 
-    $release_url = ('https://github.com/containernetworking/plugins/releases/' +
-        'download/' + $CNI_RELEASE_VERSION + '/')
+    $release_url = ('https://www.googleapis.com/storage/v1/b/gke-release/o/cni-plugins%2f' +
```
Comment (pjh) [inline cluster/gce/windows/k8s-node-setup.psm1]: Why '%2f' instead of '/' ?

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 2]
```diff
@@ -851,18 +851,18 @@ Import-Module -Name $modulePath'.replace('K8S_DIR', ${env:K8S_DIR})
 #   CLUSTER_IP_RANGE
 #   SERVICE_CLUSTER_IP_RANGE
 function Configure-CniNetworking {
-  $CNI_RELEASE_VERSION = 'v0.8.0'
+  $CNI_RELEASE_VERSION = 'v0.8.2-gke.0'
```
Comment (pjh) [inline cluster/gce/windows/k8s-node-setup.psm1]: Why -gke ? Are these different than the plugins that are released on github?

File: cluster/gce/windows/k8s-node-setup.psm1 (hunk 1 above)
Comment (yliaog) [reply cluster/gce/windows/k8s-node-setup.psm1]: it has to be encoded to be url safe. if using '/' directly here, the server would return with '404 not found'

File: cluster/gce/windows/k8s-node-setup.psm1 (hunk 2 above)
Comment (yliaog) [reply cluster/gce/windows/k8s-node-setup.psm1]: it is the same source, different build. i tested it manually, it works fine.

File: cluster/gce/windows/k8s-node-setup.psm1 (hunk 1 above)
Comment (pjh) [reply cluster/gce/windows/k8s-node-setup.psm1]: Ok - I'm not sure why the existing code seems to work <shrug>

**Comment by yliaog** [comment]
@pjh please approve

**Pull request merged**